hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -478,6 +480,15 @@ def repr_call(f, args, kwargs, *, reorder=True):
     rep = nicerepr(f)
     if rep.startswith("lambda") and ":" in rep:
         rep = f"({rep})"
+    repr_len = len(rep) + sum(len(b) for b in bits)  # approx
+    if repr_len > 30000:
```

Comment:
Number chosen to minimize complaints. Should it be lower?

Reply:
Nah, seems fine to only warn in truly egregious cases - that'd still have helped us with the problem we hit!